x/name/client/cli/gov.go

```diff
@@ -106,8 +106,94 @@ $ %s tx gov submit-proposal param-change tx gov submit-proposal \
 		},
 	}
 
-	cmd.Flags().String(FlagOwner, "", "The owner of the new name, optional (defaults to from address")
-	cmd.Flags().BoolP(flagRestricted, "r", true, "Restrict creation of child names to owner only, optional (default false)")
+	cmd.Flags().String(FlagOwner, "", "The owner of the new name, optional (defaults to from address)")
+	cmd.Flags().BoolP(flagUnrestricted, "u", false, "Allow child name creation by everyone")
+	// proposal flags
+	cmd.Flags().String(FlagTitle, "", "Title of proposal")
+	cmd.Flags().String(FlagDescription, "", "Description of proposal")
+	cmd.Flags().String(FlagDeposit, "", "Deposit of proposal")
+	return cmd
+}
+
+func GetModifyNameProposalCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "modify-name-proposal [name] (--owner [address]) (--unrestrict) [flags]",
+		Short: "Submit a modify name creation governance proposal",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Submit a modify name governance proposal along with an initial deposit.
+The proposal title and description must be provided through their respective flags.
+
+IMPORTANT: The restricted creation of sub-names will be enabled by default unless the unrestricted flag is included.
+The owner must approve all child name creation unless an alterate owner is provided.
+
+Example:
+$ %s tx gov submit-legacy-proposal \
+	modify-name-proposal \
+	<name> \
+	--unrestrict  \ 
+	--owner <key_or_address> \
```

Comment:
Nit: The keyring isn't referenced for this field, so this should just be `<address>`:
```suggestion
	--owner <address> \
```
It's also incorrectly stated as `<key_or_address>` in `GetRootNameProposalCmd`.

The `--from` field DOES use the keyring though, so it can stay as `<key_or_address>`.